.yarnrc

```diff
@@ -0,0 +1 @@
+registry "https://npm.greensock.com/"
```

Comment:
this got this private registry stuff working on my local but throws errors in the builds.  This not working as expected is why I decided to make the jump to npm as no error is thrown in that case.